examples/llama-3/sparse-finetuning.yaml

```diff
@@ -0,0 +1,77 @@
+base_model: neuralmagic/Sparse-Llama-3.1-8B-2of4
+
```

Comment:
Should we expand the example or the readme to show how to run an example e2e?
e.g start with a model --> run the integration --> save compressed?